ERCS/erc-TBA.md

```diff
@@ -0,0 +1,128 @@
+---
```

Comment:
```suggestion
---
eip: 7787
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.